src/Symfony/Component/Security/Core/Encryption/SymmetricEncryption.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Encryption;
+
+use Symfony\Component\Security\Core\Exception\DecryptionException;
+use Symfony\Component\Security\Core\Exception\MalformedCipherException;
+use Symfony\Component\Security\Core\Exception\UnsupportedAlgorithmException;
+
+class SymmetricEncryption
```

Comment:
@Nyholm what about calling this `SymmetricEncrypter`?

Reply:
I like the ...er suffix 👍🏻 it reads more natural, it really do things an is not used todo things, in this case I would prefer SymmmetricEncryption 👍🏻